ui/src/containers/ClusterMonitoring.js

```diff
@@ -40,6 +40,7 @@ const PageContainer = styled.div`
 `;
 
 const TableContainer = styled.div`
+  height: 100%;
```

Comment:
it is relative to volume?

Reply:
Nope. it's not related to the volume.  
I found this bug, it's just one line change, so want to fix it here as well.